docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -166,6 +166,15 @@ include::plugins:partial$plugin-web-app-profiles.adoc[]
 
 include::plugins:partial$plugin-web-app-steps.adoc[]
 
+=== Close browser
+
+Closes the browser and quits the remote session.
```

Comment:
What if user runs tests locally (remote session might be confusing)? I believe there is no sense to put this info